### DISCUSSION TITLE
Προθεσμίες των παραδοτέων

### DISCUSSION THREAD

**Discussion by gpbiz14** [courses-ionio/hci#1593] [Announcements]
Νομίζω ότι δεν αναφέρονται κάπου οι προθεσμίες για τα παραδοτέα. Θα μπορούσατε να μου πείτε ποιες είναι ;

**Comment by epidrome** [comment]
Η πρώτη στήλη στο πρόγραμμα του μαθήματος είναι η εβδομάδα δραστηριότητας και η παράδοση γίνεται την επόμενη εβδομάδα, προστέθηκε οδηγία κάτω από τον πίνακα:
https://github.com/courses-ionio/hci
